plugins/org.eclipse.n4js.cli/src/org/eclipse/n4js/cli/N4jscMain.java

```diff
@@ -76,10 +74,14 @@ private static N4jscOptions getOptions(String[] args) {
 		try {
 			try {
 				options.read(args);
-			} finally {
-				if (options.isVerbose()) {
-					N4jscConsole.println(options.toCallString());
+
+				if (!options.isVerbose()) {
+					// Reconfigure Logging to be quiet:
+					Logger.getRootLogger().removeAllAppenders();
+					Logger.getRootLogger().addAppender(new NullAppender());
```

Comment:
Can this be moved elsewhere? Because now the logger is being configured in a getter (i.e. major side effect in a getter)